doc/how-to-use-newconnector.md

```diff
@@ -7,7 +7,7 @@ To use the Connector type, first you need to import the sql and go-mssqldb packa
 ```
 import (
   "database/sql"
-  mssqldb "github.com/denisenkom/go-mssqldb"
+  mssql "github.com/denisenkom/go-mssqldb"
```

Comment:
But, "mssql" is the package name. The package name is *not* the last segment of the import path, it is what it is declared to be, which in this case is always "mssql", so the extra name is unneeded / undesired. Please just remove the alias.

Thanks for catching this.